src/components/Bookings/index.js

```diff
@@ -100,7 +106,7 @@ export default class Bookings extends Component {
         this.getBookings();
         this.setState({ addDetails: false, selectedbooking: {} });
       })
-      .catch(err => {});
+      .catch(err => { });
```

Comment:
you can remove the `.catch` if the anonymous function isn't going to do anything with the error.